Objects/Sensor.py

```diff
@@ -2,6 +2,27 @@
 import cv2
 from mss import mss
 from abc import ABC, abstractmethod
+import time
+
+
+import magichue
+
+
+TOKEN = 'eyJhbGciOiJSUzI1NiJ9.eyJhcHBLZXkiOiJaRzAwMSIsImNkcElkIjoiWkcwMDEiLCJjbGllbnRJZCI6Ik1ON1dBMVZZSjdBUFNHTTRNWDg3SENaNkQyMU5EWFAyIiwiZXhwaXJlRGF0ZSI6MTY5Mzc5MDkxMTY3OSwiZ3Vlc3QiOmZhbHNlLCJsb2dpbkRhdGUiOjE2NzgyMzg5MTE2NzksInJlZnJlc2hEYXRlIjoxNjg4NjA2OTExNjc5LCJzZXJ2ZXJDb2RlIjoiVVMiLCJ1aWQiOjE2MzMyMzU2ODAzOTQzMzgzMDYsInVzZXJJZCI6InNhbnRpYWdvajIyMkBnbWFpbC5jb20ifQ.n0GkAldq_xBzOgfnUskmBKWnAlSoujuzJs54WVP8KDpQ6AsJIBnvoqnxohbf-UBTrlwqlNiINT92irboO9e-fKc53dRqz-JQW3_C2FMls_lF5V10qE05fg91wfTKfvCJGdYTyqAKgV6SjCYucpb4O5DflbAlqhrAe49BE2HEw7nbGRnJsx9YMoO0ZZ6RpulF6CLqthz7qBl-0DgtQfTHecD3RJ7COvBl_qigFOvI8EHt83Mzjb77LT7dAlNW27oqC0115HFsW4duUujK4PxJPQhg9jKPY40DzTaiqYakHCZ7CkrJxGlGPScDGSk6Do5A_9CvcaYW7jRZT5LvZD2Caw'
+api = magichue.RemoteAPI.login_with_token(TOKEN)
+light = api.get_online_bulbs()[0]
+light.rgb = (0,255,0)
+
+PoliceLights = magichue.CustomMode(
```

Comment:
This is not related to a sensor. This looks like testing code, but it shouldn't live on the same file. We need to abstract the lights logic on its own class and the effects as well. This should be removed from the repository for now